### PR TITLE
UG: Add Get Started section

### PR DESCRIPTION
**Pull request by Thing1Thing2**
closing #17

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -14,25 +14,19 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `MyInsuRec.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```
Comment (rexong) [inline docs/UserGuide.md:17]: Is our jar file in `https://github.com/se-edu/addressbook-level3/releases`?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,25 +14,19 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
-
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
-
-   * **`clear`** : Deletes all contacts.
-
-   * **`exit`** : Exits the app.
+   1. view client command
+   2. add client command
+   3. delete client command
```
Comment (rexong) [inline docs/UserGuide.md:29]: Would be better if you put 

1. [`viewClient`](#viewClient):  Views a particular

In this format for all three commands with a link to the methods in the features.

**Review by rexong — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:29]: okk

File: docs/UserGuide.md (hunk 1 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:17]: no, cmi it should be <https://github.com/AY2223S1-CS2103T-W16-4/tp/releases>

File: docs/UserGuide.md [hunk 3]
```diff
@@ -24,9 +24,9 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   1. view client command
-   2. add client command
-   3. delete client command
+   1. `viewClient`: View a particular client
+   2. `addClient`: Add a client
+   3. `deleteClient`: Delete client
```
Comment (rexong) [inline docs/UserGuide.md:29]: Can consider adding a link to the method

ie

[`viewClient` ](#viewClient)

Just a suggestion, can do it sometime later. 
I will just approve this now cause I think no big issues :)

**Review by rexong — APPROVED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:29]: ok, will keep in mind to add this in next iteration.

**Pull request merged**